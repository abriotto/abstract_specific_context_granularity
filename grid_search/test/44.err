  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  9%|#################                                                                                                                                                                           | 9/99 [00:00<00:01, 81.96it/s] 18%|##################################                                                                                                                                                         | 18/99 [00:00<00:01, 65.57it/s] 25%|###############################################2                                                                                                                                           | 25/99 [00:00<00:01, 64.44it/s] 32%|############################################################4                                                                                                                              | 32/99 [00:00<00:01, 61.25it/s] 40%|###########################################################################5                                                                                                               | 40/99 [00:00<00:00, 66.05it/s] 49%|############################################################################################5                                                                                              | 49/99 [00:00<00:00, 72.88it/s] 58%|###########################################################################################################6                                                                               | 57/99 [00:00<00:00, 72.85it/s] 66%|##########################################################################################################################7                                                                | 65/99 [00:00<00:00, 66.81it/s] 73%|########################################################################################################################################                                                   | 72/99 [00:01<00:00, 65.88it/s] 81%|#######################################################################################################################################################1                                   | 80/99 [00:01<00:00, 67.74it/s] 89%|######################################################################################################################################################################2                    | 88/99 [00:01<00:00, 69.24it/s] 96%|###################################################################################################################################################################################4       | 95/99 [00:01<00:00, 67.55it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 67.60it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 32%|############################################################1                                                                                                                               | 8/25 [00:00<00:00, 63.81it/s] 68%|###############################################################################################################################1                                                           | 17/25 [00:00<00:00, 73.77it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 61.67it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 63.70it/s]
