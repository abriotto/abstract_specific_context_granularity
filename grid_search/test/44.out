# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7032799124717712, "acc": 0.5155118107795715, "length": 3.4695444107055664, "mode": "train", "epoch": 1}
{"loss": 0.6973318457603455, "acc": 0.4898148477077484, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6966177821159363, "acc": 0.5229265689849854, "length": 2.0436580181121826, "mode": "train", "epoch": 2}
{"loss": 0.6941508054733276, "acc": 0.4870370328426361, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6911714673042297, "acc": 0.5250186920166016, "length": 1.128158450126648, "mode": "train", "epoch": 3}
{"loss": 0.6934723854064941, "acc": 0.5212963819503784, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6855524182319641, "acc": 0.5472349524497986, "length": 1.2717605829238892, "mode": "train", "epoch": 4}
{"loss": 0.691980242729187, "acc": 0.5044459700584412, "length": 1.0222222805023193, "mode": "test", "epoch": 4}
{"loss": 0.6613500714302063, "acc": 0.6079941987991333, "length": 2.1231529712677, "mode": "train", "epoch": 5}
{"loss": 0.6443166136741638, "acc": 0.612797200679779, "length": 1.6666666269302368, "mode": "test", "epoch": 5}
{"loss": 0.6168510317802429, "acc": 0.6564521193504333, "length": 2.1699161529541016, "mode": "train", "epoch": 6}
{"loss": 0.6217602491378784, "acc": 0.6619501709938049, "length": 1.9333332777023315, "mode": "test", "epoch": 6}
{"loss": 0.5918638110160828, "acc": 0.676348865032196, "length": 2.128793716430664, "mode": "train", "epoch": 7}
{"loss": 0.6040202379226685, "acc": 0.6755075454711914, "length": 2.0166666507720947, "mode": "test", "epoch": 7}
{"loss": 0.5665109753608704, "acc": 0.7023298144340515, "length": 2.1398448944091797, "mode": "train", "epoch": 8}
{"loss": 0.5865995287895203, "acc": 0.7022258043289185, "length": 2.0, "mode": "test", "epoch": 8}
{"loss": 0.5442197918891907, "acc": 0.7327804565429688, "length": 2.106921672821045, "mode": "train", "epoch": 9}
{"loss": 0.5587958693504333, "acc": 0.7373052835464478, "length": 2.0166666507720947, "mode": "test", "epoch": 9}
{"loss": 0.5217909812927246, "acc": 0.7373036742210388, "length": 2.103656768798828, "mode": "train", "epoch": 10}
{"loss": 0.5485812425613403, "acc": 0.7357494831085205, "length": 2.0333333015441895, "mode": "test", "epoch": 10}
{"loss": 0.5023635625839233, "acc": 0.7480413317680359, "length": 2.147280693054199, "mode": "train", "epoch": 11}
{"loss": 0.5335344672203064, "acc": 0.7375646829605103, "length": 2.0166666507720947, "mode": "test", "epoch": 11}
{"loss": 0.497918039560318, "acc": 0.7526661157608032, "length": 2.200380563735962, "mode": "train", "epoch": 12}
{"loss": 0.5242866277694702, "acc": 0.7421708703041077, "length": 2.0166666507720947, "mode": "test", "epoch": 12}
{"loss": 0.4848826825618744, "acc": 0.7510175704956055, "length": 2.2020063400268555, "mode": "train", "epoch": 13}
{"loss": 0.5221740007400513, "acc": 0.7422009110450745, "length": 2.0, "mode": "test", "epoch": 13}
{"loss": 0.4984413683414459, "acc": 0.7505350708961487, "length": 2.1744351387023926, "mode": "train", "epoch": 14}
{"loss": 0.5241414308547974, "acc": 0.7391048073768616, "length": 1.9666666984558105, "mode": "test", "epoch": 14}
{"loss": 0.48630577325820923, "acc": 0.7584646940231323, "length": 2.2396187782287598, "mode": "train", "epoch": 15}
{"loss": 0.527070939540863, "acc": 0.7371083498001099, "length": 1.9333332777023315, "mode": "test", "epoch": 15}
{"loss": 0.48297178745269775, "acc": 0.7549203634262085, "length": 2.207829236984253, "mode": "train", "epoch": 16}
{"loss": 0.5459863543510437, "acc": 0.7431545853614807, "length": 2.1500000953674316, "mode": "test", "epoch": 16}
{"loss": 0.47212734818458557, "acc": 0.7734407782554626, "length": 2.4111499786376953, "mode": "train", "epoch": 17}
{"loss": 0.5150057077407837, "acc": 0.7396491169929504, "length": 2.116666555404663, "mode": "test", "epoch": 17}
{"loss": 0.46162864565849304, "acc": 0.7785618305206299, "length": 2.603663682937622, "mode": "train", "epoch": 18}
{"loss": 0.5181318521499634, "acc": 0.7528979778289795, "length": 2.316666603088379, "mode": "test", "epoch": 18}
{"loss": 0.44362446665763855, "acc": 0.7930266261100769, "length": 2.7278101444244385, "mode": "train", "epoch": 19}
{"loss": 0.5069571137428284, "acc": 0.7737447023391724, "length": 2.9666666984558105, "mode": "test", "epoch": 19}
{"loss": 0.4307823181152344, "acc": 0.8106256127357483, "length": 2.931206703186035, "mode": "train", "epoch": 20}
{"loss": 0.46051639318466187, "acc": 0.7899027466773987, "length": 2.866666555404663, "mode": "test", "epoch": 20}
{"loss": 0.4176025986671448, "acc": 0.8186114430427551, "length": 3.1367573738098145, "mode": "train", "epoch": 21}
{"loss": 0.4561881422996521, "acc": 0.8003472089767456, "length": 2.816666603088379, "mode": "test", "epoch": 21}
{"loss": 0.39722248911857605, "acc": 0.8336263298988342, "length": 3.114982843399048, "mode": "train", "epoch": 22}
{"loss": 0.4553660750389099, "acc": 0.8081030249595642, "length": 3.2166666984558105, "mode": "test", "epoch": 22}
{"loss": 0.37991681694984436, "acc": 0.8406215906143188, "length": 3.244431495666504, "mode": "train", "epoch": 23}
{"loss": 0.4350430965423584, "acc": 0.8153857588768005, "length": 3.049999952316284, "mode": "test", "epoch": 23}
{"loss": 0.3707464635372162, "acc": 0.841310441493988, "length": 3.2069263458251953, "mode": "train", "epoch": 24}
{"loss": 0.44368553161621094, "acc": 0.8204371929168701, "length": 3.049999952316284, "mode": "test", "epoch": 24}
{"loss": 0.370266854763031, "acc": 0.8406569957733154, "length": 3.2204668521881104, "mode": "train", "epoch": 25}
{"loss": 0.4296199083328247, "acc": 0.8250290155410767, "length": 3.0, "mode": "test", "epoch": 25}
{"loss": 0.3532610535621643, "acc": 0.8496434092521667, "length": 3.204097032546997, "mode": "train", "epoch": 26}
{"loss": 0.45911961793899536, "acc": 0.8214163780212402, "length": 3.1666667461395264, "mode": "test", "epoch": 26}
{"loss": 0.3621479570865631, "acc": 0.8530026078224182, "length": 3.3320066928863525, "mode": "train", "epoch": 27}
{"loss": 0.4141206741333008, "acc": 0.8326656818389893, "length": 3.0833332538604736, "mode": "test", "epoch": 27}
{"loss": 0.35705581307411194, "acc": 0.8570621013641357, "length": 3.22579288482666, "mode": "train", "epoch": 28}
{"loss": 0.4496995806694031, "acc": 0.8230224847793579, "length": 3.200000047683716, "mode": "test", "epoch": 28}
{"loss": 0.34360042214393616, "acc": 0.8509496450424194, "length": 3.271066665649414, "mode": "train", "epoch": 29}
{"loss": 0.422311007976532, "acc": 0.83009272813797, "length": 3.116666555404663, "mode": "test", "epoch": 29}
{"loss": 0.3448749780654907, "acc": 0.8613824844360352, "length": 3.3479645252227783, "mode": "train", "epoch": 30}
{"loss": 0.41869521141052246, "acc": 0.8224759697914124, "length": 3.2333333492279053, "mode": "test", "epoch": 30}
{"loss": 0.3310790956020355, "acc": 0.8623125553131104, "length": 3.3340847492218018, "mode": "train", "epoch": 31}
{"loss": 0.4190102517604828, "acc": 0.8321614861488342, "length": 3.3499999046325684, "mode": "test", "epoch": 31}
{"loss": 0.3221859037876129, "acc": 0.8705119490623474, "length": 3.4439351558685303, "mode": "train", "epoch": 32}
{"loss": 0.4024946391582489, "acc": 0.8370781540870667, "length": 3.327777862548828, "mode": "test", "epoch": 32}
{"loss": 0.32141175866127014, "acc": 0.8682363629341125, "length": 3.3936243057250977, "mode": "train", "epoch": 33}
{"loss": 0.43698614835739136, "acc": 0.8199307918548584, "length": 3.3333332538604736, "mode": "test", "epoch": 33}
{"loss": 0.32315701246261597, "acc": 0.8731610774993896, "length": 3.451878070831299, "mode": "train", "epoch": 34}
{"loss": 0.3972039222717285, "acc": 0.8408231735229492, "length": 3.305555582046509, "mode": "test", "epoch": 34}
{"loss": 0.31125608086586, "acc": 0.872083842754364, "length": 3.46364164352417, "mode": "train", "epoch": 35}
{"loss": 0.4024932086467743, "acc": 0.8471989035606384, "length": 3.4111111164093018, "mode": "test", "epoch": 35}
{"loss": 0.3048739731311798, "acc": 0.877358615398407, "length": 3.4870386123657227, "mode": "train", "epoch": 36}
{"loss": 0.3965117335319519, "acc": 0.8257078528404236, "length": 3.3499999046325684, "mode": "test", "epoch": 36}
{"loss": 0.2946530878543854, "acc": 0.8813725709915161, "length": 3.5390913486480713, "mode": "train", "epoch": 37}
{"loss": 0.405122846364975, "acc": 0.8416711688041687, "length": 3.4277777671813965, "mode": "test", "epoch": 37}
{"loss": 0.3000063896179199, "acc": 0.8747372031211853, "length": 3.486393690109253, "mode": "train", "epoch": 38}
{"loss": 0.40107855200767517, "acc": 0.8457554578781128, "length": 3.4555554389953613, "mode": "test", "epoch": 38}
{"loss": 0.28980752825737, "acc": 0.8805088996887207, "length": 3.535794734954834, "mode": "train", "epoch": 39}
{"loss": 0.3938058614730835, "acc": 0.8490707278251648, "length": 3.5222222805023193, "mode": "test", "epoch": 39}
{"loss": 0.2945839464664459, "acc": 0.8787930011749268, "length": 3.5202646255493164, "mode": "train", "epoch": 40}
{"loss": 0.42046305537223816, "acc": 0.8459401726722717, "length": 3.3722221851348877, "mode": "test", "epoch": 40}
{"loss": 0.2964283227920532, "acc": 0.8814989328384399, "length": 3.511716604232788, "mode": "train", "epoch": 41}
{"loss": 0.38432377576828003, "acc": 0.8487891554832458, "length": 3.4833333492279053, "mode": "test", "epoch": 41}
{"loss": 0.2900710701942444, "acc": 0.8826961517333984, "length": 3.5351719856262207, "mode": "train", "epoch": 42}
{"loss": 0.4101055860519409, "acc": 0.8503907322883606, "length": 3.4777777194976807, "mode": "test", "epoch": 42}
{"loss": 0.28361445665359497, "acc": 0.8845382332801819, "length": 3.4728550910949707, "mode": "train", "epoch": 43}
{"loss": 0.40190523862838745, "acc": 0.8496817946434021, "length": 3.5, "mode": "test", "epoch": 43}
{"loss": 0.277579128742218, "acc": 0.8904906511306763, "length": 3.509843349456787, "mode": "train", "epoch": 44}
{"loss": 0.41933584213256836, "acc": 0.848706841468811, "length": 3.538888931274414, "mode": "test", "epoch": 44}
{"loss": 0.27865666151046753, "acc": 0.8884570598602295, "length": 3.4616472721099854, "mode": "train", "epoch": 45}
{"loss": 0.3928300738334656, "acc": 0.8543592095375061, "length": 3.5, "mode": "test", "epoch": 45}
{"loss": 0.2767750322818756, "acc": 0.8855865597724915, "length": 3.54526686668396, "mode": "train", "epoch": 46}
{"loss": 0.41139787435531616, "acc": 0.8536547422409058, "length": 3.683333396911621, "mode": "test", "epoch": 46}
{"loss": 0.26814618706703186, "acc": 0.889687716960907, "length": 3.5879271030426025, "mode": "train", "epoch": 47}
{"loss": 0.3972907066345215, "acc": 0.8529892563819885, "length": 3.6500000953674316, "mode": "test", "epoch": 47}
{"loss": 0.27250561118125916, "acc": 0.8925279378890991, "length": 3.610846757888794, "mode": "train", "epoch": 48}
{"loss": 0.42905592918395996, "acc": 0.8296464085578918, "length": 3.6666667461395264, "mode": "test", "epoch": 48}
{"loss": 0.2762559652328491, "acc": 0.8913902640342712, "length": 3.639705181121826, "mode": "train", "epoch": 49}
{"loss": 0.40452682971954346, "acc": 0.8537849187850952, "length": 3.683333396911621, "mode": "test", "epoch": 49}
{"loss": 0.26150503754615784, "acc": 0.8944132328033447, "length": 3.628437042236328, "mode": "train", "epoch": 50}
{"loss": 0.42060160636901855, "acc": 0.8469573259353638, "length": 3.7166666984558105, "mode": "test", "epoch": 50}
{"loss": 0.26046258211135864, "acc": 0.8921626806259155, "length": 3.5871999263763428, "mode": "train", "epoch": 51}
{"loss": 0.40608078241348267, "acc": 0.8472400307655334, "length": 3.799999952316284, "mode": "test", "epoch": 51}
{"loss": 0.2486303448677063, "acc": 0.895889937877655, "length": 3.6915476322174072, "mode": "train", "epoch": 52}
{"loss": 0.4171743094921112, "acc": 0.8384292721748352, "length": 3.75, "mode": "test", "epoch": 52}
{"loss": 0.2519773542881012, "acc": 0.896744966506958, "length": 3.6777334213256836, "mode": "train", "epoch": 53}
{"loss": 0.4390222430229187, "acc": 0.8349982500076294, "length": 3.866666555404663, "mode": "test", "epoch": 53}
{"loss": 0.2517746686935425, "acc": 0.8991009593009949, "length": 3.6981756687164307, "mode": "train", "epoch": 54}
{"loss": 0.4129025340080261, "acc": 0.8529781699180603, "length": 3.7166666984558105, "mode": "test", "epoch": 54}
{"loss": 0.24495945870876312, "acc": 0.902062177658081, "length": 3.6012015342712402, "mode": "train", "epoch": 55}
{"loss": 0.4150737226009369, "acc": 0.8612257242202759, "length": 3.683333396911621, "mode": "test", "epoch": 55}
{"loss": 0.23797494173049927, "acc": 0.9042341709136963, "length": 3.627838134765625, "mode": "train", "epoch": 56}
{"loss": 0.3771323561668396, "acc": 0.865877628326416, "length": 3.9166667461395264, "mode": "test", "epoch": 56}
{"loss": 0.23993831872940063, "acc": 0.9050500988960266, "length": 3.715672492980957, "mode": "train", "epoch": 57}
{"loss": 0.39783066511154175, "acc": 0.8577858209609985, "length": 3.8333332538604736, "mode": "test", "epoch": 57}
{"loss": 0.23993569612503052, "acc": 0.9058265089988708, "length": 3.6660404205322266, "mode": "train", "epoch": 58}
{"loss": 0.38059934973716736, "acc": 0.8670139908790588, "length": 3.816666603088379, "mode": "test", "epoch": 58}
{"loss": 0.23983345925807953, "acc": 0.9022378921508789, "length": 3.639216661453247, "mode": "train", "epoch": 59}
{"loss": 0.3556021451950073, "acc": 0.8583389520645142, "length": 3.883333444595337, "mode": "test", "epoch": 59}
{"loss": 0.23096752166748047, "acc": 0.9077032208442688, "length": 3.7100632190704346, "mode": "train", "epoch": 60}
{"loss": 0.35127633810043335, "acc": 0.8700476884841919, "length": 3.883333444595337, "mode": "test", "epoch": 60}
{"loss": 0.23832716047763824, "acc": 0.9059356451034546, "length": 3.694309949874878, "mode": "train", "epoch": 61}
{"loss": 0.3627074658870697, "acc": 0.8634370565414429, "length": 3.816666603088379, "mode": "test", "epoch": 61}
{"loss": 0.23587264120578766, "acc": 0.9007357358932495, "length": 3.6847310066223145, "mode": "train", "epoch": 62}
{"loss": 0.36282879114151, "acc": 0.8753672242164612, "length": 3.883333444595337, "mode": "test", "epoch": 62}
{"loss": 0.22354432940483093, "acc": 0.9154542088508606, "length": 3.686203956604004, "mode": "train", "epoch": 63}
{"loss": 0.3487030863761902, "acc": 0.8812099695205688, "length": 3.9000000953674316, "mode": "test", "epoch": 63}
{"loss": 0.22026221454143524, "acc": 0.9199320673942566, "length": 3.690380811691284, "mode": "train", "epoch": 64}
{"loss": 0.37005698680877686, "acc": 0.8660690188407898, "length": 3.7666666507720947, "mode": "test", "epoch": 64}
{"loss": 0.2156469076871872, "acc": 0.9135478138923645, "length": 3.607372999191284, "mode": "train", "epoch": 65}
{"loss": 0.3851484954357147, "acc": 0.8724203109741211, "length": 3.883333444595337, "mode": "test", "epoch": 65}
{"loss": 0.20819073915481567, "acc": 0.9196552038192749, "length": 3.6738266944885254, "mode": "train", "epoch": 66}
{"loss": 0.35511285066604614, "acc": 0.8729411363601685, "length": 3.8499999046325684, "mode": "test", "epoch": 66}
{"loss": 0.20546670258045197, "acc": 0.9181709885597229, "length": 3.6367604732513428, "mode": "train", "epoch": 67}
{"loss": 0.3444797396659851, "acc": 0.8788929581642151, "length": 3.8499999046325684, "mode": "test", "epoch": 67}
{"loss": 0.2093876153230667, "acc": 0.9109293818473816, "length": 3.6769509315490723, "mode": "train", "epoch": 68}
{"loss": 0.3516804575920105, "acc": 0.8850760459899902, "length": 3.883333444595337, "mode": "test", "epoch": 68}
{"loss": 0.20306555926799774, "acc": 0.9190646409988403, "length": 3.6914052963256836, "mode": "train", "epoch": 69}
{"loss": 0.3306753635406494, "acc": 0.8865385055541992, "length": 3.883333444595337, "mode": "test", "epoch": 69}
{"loss": 0.2182014286518097, "acc": 0.916755199432373, "length": 3.675808906555176, "mode": "train", "epoch": 70}
{"loss": 0.3661055564880371, "acc": 0.8660300970077515, "length": 3.8333332538604736, "mode": "test", "epoch": 70}
{"loss": 0.23119153082370758, "acc": 0.9074200391769409, "length": 3.6329433917999268, "mode": "train", "epoch": 71}
{"loss": 0.34816429018974304, "acc": 0.877575159072876, "length": 3.9166667461395264, "mode": "test", "epoch": 71}
{"loss": 0.2122555524110794, "acc": 0.9151737093925476, "length": 3.7359707355499268, "mode": "train", "epoch": 72}
{"loss": 0.33622750639915466, "acc": 0.8803362250328064, "length": 3.9000000953674316, "mode": "test", "epoch": 72}
{"loss": 0.2043324112892151, "acc": 0.9194648861885071, "length": 3.6931800842285156, "mode": "train", "epoch": 73}
{"loss": 0.35220688581466675, "acc": 0.8758215308189392, "length": 3.883333444595337, "mode": "test", "epoch": 73}
{"loss": 0.19731402397155762, "acc": 0.9231850504875183, "length": 3.7100470066070557, "mode": "train", "epoch": 74}
{"loss": 0.38129159808158875, "acc": 0.8780192732810974, "length": 3.933333396911621, "mode": "test", "epoch": 74}
{"loss": 0.2016942799091339, "acc": 0.920552134513855, "length": 3.730731964111328, "mode": "train", "epoch": 75}
{"loss": 0.36589789390563965, "acc": 0.8800125122070312, "length": 3.866666555404663, "mode": "test", "epoch": 75}
{"loss": 0.2102113515138626, "acc": 0.9196159243583679, "length": 3.6953911781311035, "mode": "train", "epoch": 76}
{"loss": 0.36336368322372437, "acc": 0.8769776821136475, "length": 3.844444513320923, "mode": "test", "epoch": 76}
{"loss": 0.19123147428035736, "acc": 0.9246843457221985, "length": 3.6448113918304443, "mode": "train", "epoch": 77}
{"loss": 0.3553165793418884, "acc": 0.877877950668335, "length": 3.9166667461395264, "mode": "test", "epoch": 77}
{"loss": 0.19429130852222443, "acc": 0.921951413154602, "length": 3.708991050720215, "mode": "train", "epoch": 78}
{"loss": 0.3589157462120056, "acc": 0.877136766910553, "length": 3.933333396911621, "mode": "test", "epoch": 78}
{"loss": 0.18686212599277496, "acc": 0.9290341734886169, "length": 3.690568447113037, "mode": "train", "epoch": 79}
{"loss": 0.3330584764480591, "acc": 0.8904981017112732, "length": 3.855555534362793, "mode": "test", "epoch": 79}
{"loss": 0.18131659924983978, "acc": 0.9317759275436401, "length": 3.6266210079193115, "mode": "train", "epoch": 80}
{"loss": 0.36344966292381287, "acc": 0.8836471438407898, "length": 3.883333444595337, "mode": "test", "epoch": 80}
{"loss": 0.1778387725353241, "acc": 0.9319962859153748, "length": 3.708845853805542, "mode": "train", "epoch": 81}
{"loss": 0.33444854617118835, "acc": 0.8822706341743469, "length": 3.8722221851348877, "mode": "test", "epoch": 81}
{"loss": 0.1783684641122818, "acc": 0.9345922470092773, "length": 3.69734263420105, "mode": "train", "epoch": 82}
{"loss": 0.37260377407073975, "acc": 0.8766281008720398, "length": 3.883333444595337, "mode": "test", "epoch": 82}
{"loss": 0.1961493045091629, "acc": 0.9247968196868896, "length": 3.651461124420166, "mode": "train", "epoch": 83}
{"loss": 0.3612898588180542, "acc": 0.8825498819351196, "length": 3.844444513320923, "mode": "test", "epoch": 83}
{"loss": 0.18377159535884857, "acc": 0.9278576374053955, "length": 3.6409764289855957, "mode": "train", "epoch": 84}
{"loss": 0.3639254570007324, "acc": 0.8692874312400818, "length": 3.855555534362793, "mode": "test", "epoch": 84}
{"loss": 0.19751428067684174, "acc": 0.9267523288726807, "length": 3.6958742141723633, "mode": "train", "epoch": 85}
{"loss": 0.3662134110927582, "acc": 0.8811376094818115, "length": 3.8888888359069824, "mode": "test", "epoch": 85}
{"loss": 0.1903391033411026, "acc": 0.9299898147583008, "length": 3.724592447280884, "mode": "train", "epoch": 86}
{"loss": 0.35188740491867065, "acc": 0.8768441081047058, "length": 3.9000000953674316, "mode": "test", "epoch": 86}
{"loss": 0.1853732317686081, "acc": 0.93287593126297, "length": 3.6725780963897705, "mode": "train", "epoch": 87}
{"loss": 0.3529684543609619, "acc": 0.8787994980812073, "length": 3.883333444595337, "mode": "test", "epoch": 87}
{"loss": 0.16828516125679016, "acc": 0.9368894100189209, "length": 3.6740994453430176, "mode": "train", "epoch": 88}
{"loss": 0.3487031161785126, "acc": 0.8889268040657043, "length": 3.866666555404663, "mode": "test", "epoch": 88}
{"loss": 0.17265965044498444, "acc": 0.9355902671813965, "length": 3.626447916030884, "mode": "train", "epoch": 89}
{"loss": 0.36892926692962646, "acc": 0.8750913739204407, "length": 3.711111068725586, "mode": "test", "epoch": 89}
{"loss": 0.1835215836763382, "acc": 0.9298787117004395, "length": 3.604804754257202, "mode": "train", "epoch": 90}
{"loss": 0.38130539655685425, "acc": 0.8752415180206299, "length": 3.8333332538604736, "mode": "test", "epoch": 90}
{"loss": 0.1647532880306244, "acc": 0.9355962872505188, "length": 3.6691410541534424, "mode": "train", "epoch": 91}
{"loss": 0.3598667085170746, "acc": 0.8817642331123352, "length": 3.894444465637207, "mode": "test", "epoch": 91}
{"loss": 0.16343288123607635, "acc": 0.9351473450660706, "length": 3.7451844215393066, "mode": "train", "epoch": 92}
{"loss": 0.3533781170845032, "acc": 0.8881410360336304, "length": 3.883333444595337, "mode": "test", "epoch": 92}
{"loss": 0.16326768696308136, "acc": 0.9385170340538025, "length": 3.7076518535614014, "mode": "train", "epoch": 93}
{"loss": 0.36262279748916626, "acc": 0.890602707862854, "length": 3.883333444595337, "mode": "test", "epoch": 93}
{"loss": 0.15534663200378418, "acc": 0.9421690702438354, "length": 3.6771559715270996, "mode": "train", "epoch": 94}
{"loss": 0.3627999424934387, "acc": 0.8963785767555237, "length": 3.9000000953674316, "mode": "test", "epoch": 94}
{"loss": 0.1545863002538681, "acc": 0.9372158646583557, "length": 3.7081451416015625, "mode": "train", "epoch": 95}
{"loss": 0.3603052496910095, "acc": 0.8904958367347717, "length": 3.883333444595337, "mode": "test", "epoch": 95}
{"loss": 0.15851561725139618, "acc": 0.9398953914642334, "length": 3.7153396606445312, "mode": "train", "epoch": 96}
{"loss": 0.35727983713150024, "acc": 0.8975004553794861, "length": 3.844444513320923, "mode": "test", "epoch": 96}
{"loss": 0.15918514132499695, "acc": 0.9339627027511597, "length": 3.6371450424194336, "mode": "train", "epoch": 97}
{"loss": 0.3695903718471527, "acc": 0.8877726197242737, "length": 3.844444513320923, "mode": "test", "epoch": 97}
{"loss": 0.1604466736316681, "acc": 0.9361762404441833, "length": 3.7029662132263184, "mode": "train", "epoch": 98}
{"loss": 0.367519348859787, "acc": 0.8753606081008911, "length": 3.8222222328186035, "mode": "test", "epoch": 98}
{"loss": 0.16687019169330597, "acc": 0.932194173336029, "length": 3.687410593032837, "mode": "train", "epoch": 99}
{"loss": 0.4018319547176361, "acc": 0.8842281699180603, "length": 3.788888931274414, "mode": "test", "epoch": 99}
{"loss": 0.16850696504116058, "acc": 0.9366940259933472, "length": 3.6188578605651855, "mode": "train", "epoch": 100}
{"loss": 0.3673650622367859, "acc": 0.8825487494468689, "length": 3.7166666984558105, "mode": "test", "epoch": 100}
test accuracy: 0.7611294388771057
